tests/symbolicator/snapshots/SymbolicatorMinidumpIntegrationTest/test_reprocessing_reprocessed.pysnap

```diff
@@ -14,6 +14,16 @@ contexts:
 debug_meta:
   images:
   - arch: x86
+    candidates:
+    - download:
+        features:
+          has_debug_info: true
+          has_sources: false
+          has_symbols: true
+          has_unwind_info: true
+        status: ok
+      location: '4'
```

Comment:
Are these db ids stable? If not, please remove or mask them before passing them into the snapshot.